assets/src/scripts/components/hydrograph/scales.js

```diff
@@ -75,7 +76,14 @@ export const getXScale = memoize((kind, tsKey) => createSelector(
         if (kind === 'BRUSH') {
             timeRange = requestTimeRange;
         } else {
-            timeRange = hydrographXRange ? hydrographXRange : requestTimeRange;
+            if (hydrographXRange && Object.keys(hydrographXRange).length > 0) {
```

Comment:
This seems like we are fixing a symptom instead of the problem. The issue is that getRequestTimeRange appears to be returning the wrong range. The comment of getRequestTimeRange  is 
```
 @return {Object} with start and end {Number} properties that contain the range of the data requested in universal time or null if the store does not contain a query for the tsKey request```